snowplow/iglu.json

```diff
@@ -6,14 +6,22 @@
       {
         "name": "Iglu Central",
         "priority": 0,
-        "vendorPrefixes": [
-          "com.snowplowanalytics"
-        ],
+        "vendorPrefixes": [ "com.snowplowanalytics" ],
         "connection": {
           "http": {
             "uri": "http://iglucentral.com"
           }
         }
+      },
+      {
+        "name": "local Iglu repository",
+        "priority": 10,
+        "vendorPrefixes": [ "com.snowplowanalytics" ],
+        "connection": {
+          "http": {
+            "uri": "http://iglu:80"
+          }
```

Comment:
**Question (not-blocking):** Curious why we added this. Does this allow some better local testing?

Reply:
Yes, previously I wasn't able to test the new event with a schema that just existed locally, and hadn't been published yet to SnowCatCloud.